README.md

```diff
@@ -238,48 +142,24 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project, give it a ⭐
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
-
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+I would like to thank
 
-I would like to thank...
+- [Microverse](https://www.microverse.org/)
+- [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+- [CoinCap](https://docs.coincap.io/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] Your README is well documented 💯 I just noticed that the MIT link is broken, so I suggest to fix it by changing the path from `[MIT](./LICENSE)` to `[MIT](./MIT.md)` 👍

> And remember to customize your MIT.md file 💡
![image](https://user-images.githubusercontent.com/45231276/204957137-4dedb378-191e-4d8a-8a19-10e28f128bae.png)

Reply:
Thank you for the constructive feedback.
Issues fix!

Reply:
Thank you for the constructive feedback.
Issues fix!